docker-hub/publish/insights-analytics.md

```diff
@@ -47,84 +48,70 @@ The HTTP API endpoints are available at:
 using the API in the [DVP Data API documentation](/docker-hub/api/dvp/){:
 target="_blank" rel="noopener" class="_"}.
 
-## Data formats
+## Data points
 
-The data can be exported in either raw or summary format. Each format contains
-different data points and are formatted differently.
+Export data in either raw or summary format. Each format contains different data
+points and with different structure.
 
-Review the [Data definitions](#data-definitions) section for more information
-about the data points and how to read them.
+The following sections describe the available data points for each format. The
+**Available from** column shows when the field was first added.
 
 ### Raw data
 
-The raw data format contains the following data points for the selected time
-span. Each action is represented as a single row in the CSV file.
-
-- Timestamp
-- Namespace
-- Repository
-- Reference
-- Digest
-- Tag (included when available)
-- Action day
-- HTTP method
-- Action, one of the following:
-  - Pull by tag
-  - Pull by digest
-  - Version check
-- Type
-- Host
-- Country
-- User agent tool
-- User agent version
+The raw data format contains the following data points. Each row in the CSV file
+represents an image pull.
+
+| Data point                    | Value                                             | Description                                                                           | Available from  |
+| ----------------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------- | --------------- |
+| Timestamp                     | `YYYY-MM-DD 00:00:00`                             | Date and time of the request.                                                         | January 1, 2022 |
+| Namespace                     | String                                            | Docker organization (image namespace).                                                | January 1, 2022 |
+| Repository                    | String                                            | Image name.                                                                           | January 1, 2022 |
+| Reference                     | String                                            | Image digest or tag, as requested.                                                    | January 1, 2022 |
+| Digest                        | String                                            | Image digest.                                                                         | January 1, 2022 |
```

Comment:
**suggestion (non-blocking):** I feel that these self-explanatory descriptions are unnecessary 🤔 

I think we could do one of two things for these cases:
- Include an example value
- Include a link to a documentation page about the property

Or something else, but essentially making the most of this space.

What do you think?

Reply:
yeah, my idea was to try and make it easier to get an idea of the value format _at a glance_, but I agree it seems more strange than useful. I've tried joining the value and description columns together again